package day15;

import java.util.Scanner;

public class Exception02 {

	public static void main(String[] args) {
		//try~catch문으로 예외처리

	}
	//두 수, 연사자를 입력받아 4칙연산을 수행하고 결과를 리턴하는 메서드 생성
	Scanner scan = new Scanner(System.in);
	int num1 = scan.nextInt();
	int num2 = scan.nextInt();
	char a = scan.next().charAt(0);
	
	switch(a) {
	case '+' : System.out.println(num1+"+"+num2+"="+num1+num2); break;
	
	}

}
